platforms/hyperledger-fabric/configuration/roles/create/crypto/orderer/tasks/orderer.yaml

```diff
@@ -10,7 +10,7 @@
   changed_when: false
   tags:
     - no-idempotence-test
-
+  
```

Comment:
The following task should be changed as not vault_msp_result will have only one response. Please add a new task to get the complete msp secret and then pass that variable in vault_output for setup/get_crypto role